torchvision/prototype/datasets/_builtin/caltech.py

```diff
@@ -0,0 +1,209 @@
+import io
+import pathlib
+from typing import Any, Callable, Dict, List, Optional, Tuple, Union
+import re
+
+import numpy as np
```

Comment:
Do we need numpy here? You only use it to cast the array to `int64`, which you can also do in numpy with a string (or directly add a `dtype` in `torch.as_tensor` / `torch.tensor`)

Reply:
Unfortunately, we do. The `box_coord` array has dtype uint16, which `torch.as_tensor` cannot handle. So either we use numpy to convert to a common dtype first, or we could also convert the values to a list and build a tensor from that. What would you prefer? In any case I'll add a comment explaining what is going on.